Containerfile

```diff
@@ -1,4 +1,5 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+#FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+FROM quay.io/centos/centos:stream9-minimal
```

Comment:
This is probably why EC is failing. Needed for `sysstat`?

Reply:
Yeah I've added the Centos 9 stream repository instead in the latest version